src/JSInterop/Microsoft.JSInterop/src/IJSUnmarshalledRuntime.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.JSInterop
+{
+    /// <summary>
+    /// Represents an instance of a JavaScript runtime to which calls may be dispatched without JSON marshalling.
+    /// Not all JavaScript runtimes support this capability. Currently it is only supported on WebAssembly and for
+    /// security reasons, will never be supported for .NET code that runs on the server.
+    /// This is an advanced mechanism that should only be used in performance-critical scenarios.
+    /// </summary>
+    public interface IJSUnmarshalledRuntime
```

Comment:
Could we mark this as `[SupportedOSPlatform('browser")]`? Previously you were unable to get to this contract from a Blazor server app. We have a way to indicate this is never useful in it.